### PR TITLE
Improve branding for our extension

### PR DESCRIPTION
**Pull request by rynowak**
/cc @eilon

### CODE REVIEW

**Comment by rynowak** [comment]
image

**Comment by rynowak** [comment]
We need to fix the version number too.

File: tooling/Microsoft.VisualStudio.RazorExtension/RazorPackage.vsct [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <Parent guid="guidSHLMainMenu" id="IDG_VS_WNDO_OTRWNDWS1"/>
         <Icon guid="guidImages" id="bmpPic1" />
         <Strings>
-          <ButtonText>RazorInfoToolWindow</ButtonText>
+          <ButtonText>Razor Diagnostics</ButtonText>
```
Comment (rynowak) [inline tooling/Microsoft.VisualStudio.RazorExtension/RazorPackage.vsct]: This won't show up anywhere in what we're shipping. I just wanted to make it less nasty.

File: tooling/Microsoft.VisualStudio.RazorExtension/VSPackage.resx [hunk 2]
```diff
@@ -127,13 +117,13 @@
...
   </resheader>
-  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="110" xml:space="preserve">
-    <value>RazorPackage Extension</value>
+    <value>ASP.NET Core Razor Language Services</value>
   </data>
   <data name="112" xml:space="preserve">
-    <value>RazorPackage Visual Studio Extension Detailed Info</value>
+    <value>Provides languages services for ASP.NET Core Razor.</value>
   </data>
```
Comment (rynowak) [inline tooling/Microsoft.VisualStudio.RazorExtension/VSPackage.resx]: These are used in the about dialog.

**Comment by rynowak** [comment]
Eilon signed off in persons

**Comment by rynowak** [comment]
f0d5461e3158e9ef37006750a18c72b0c2770fcf

File: tooling/Microsoft.VisualStudio.RazorExtension/VSPackage.resx [hunk 3]
```diff
@@ -127,13 +117,13 @@
...
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="110" xml:space="preserve">
-    <value>RazorPackage Extension</value>
+    <value>ASP.NET Core Razor Language Services</value>
   </data>
   <data name="112" xml:space="preserve">
-    <value>RazorPackage Visual Studio Extension Detailed Info</value>
+    <value>Provides languages services for ASP.NET Core Razor.</value>
```
Comment (Eilon) [inline tooling/Microsoft.VisualStudio.RazorExtension/VSPackage.resx]: Derp derp, was `languages` meant to be pluralized here?

**Pull request closed without merging**